demo/index.html

```diff
@@ -15,17 +15,17 @@
     <main></main>
 
     <script type="module">
-      import 'marked/marked.min.js';
-      import 'prismjs';
+      import 'https://unpkg.com/marked@latest/marked.min.js';
+      import 'https://unpkg.com/prismjs@latest/prism.js';
       fetch('/demo/demo.md')
         .then((response) => response.text())
         .then((text) => {
-          const rawHtml = marked(text);
+          const rawHtml = marked.parse(text);
```

Comment:
The `npm run serve` was failing to find `marked` from the local dependency before and I noticed that this is how the packages are loaded in the [auro-hyperlink](https://github.com/AlaskaAirlines/auro-hyperlink/blob/main/demo/index.html#L33) so I thought it would make sense to update it.

Reply:
I suspect here that the issue is just that the marked dependency needs to be updated in the package.json and then line 23 updated. You should be able to do that and then ditch the changes on 18 and 19.

Reply:
Here's what I'm seeing after updating the marked dependency: 

## Browser
![image](https://user-images.githubusercontent.com/23248722/229204078-9992bbd3-db24-4ed1-80d4-ab97535f6e65.png)

## Terminal
![image](https://user-images.githubusercontent.com/23248722/229204137-ee0823a6-3efe-41df-828d-84a7c50d32b2.png)

## package.json
![image](https://user-images.githubusercontent.com/23248722/229204230-daf02677-cc9b-4857-9d13-3bb2f2ba9dd5.png)

Reply:
Using the CDN is something that we have been doing for a long time now. All repos that are to be updated should be doing the same thing. 

https://github.com/AlaskaAirlines/WC-Generator/blame/master/template/demo/index.html